src/patternfly/components/Wizard/wizard.scss

```diff
@@ -115,9 +119,6 @@
   --pf-c-wizard__nav-link--m-disabled--Color: var(--pf-global--Color--dark-200);
 
   // Nav link number
-  --pf-c-wizard__nav-link--before--Width: 1.5rem;
-  --pf-c-wizard__nav-link--before--Height: 1.5rem;
-  --pf-c-wizard__nav-link--before--Top: 0;
```

Comment:
instead of moving just these 3 lines, can you move this whole block up so that the `// Nav link number` CSS is all in one spot?